.github/workflows/release-windows.yml

```diff
@@ -156,15 +145,19 @@ jobs:
           -DHOST_BINARY_DIR=${GITHUB_WORKSPACE}/build-host \
           -DCMAKE_BUILD_TYPE=RelWithDebInfo \
           -DOPTION_SURVEY_KEY=${{ inputs.survey_key }} \
-          -DWINDOWS_CERTIFICATE_COMMON_NAME="${WINDOWS_CERTIFICATE_COMMON_NAME}" \
           # EOF
         echo "::endgroup::"
 
         echo "::group::Build"
         cmake --build . --target openttd
         echo "::endgroup::"
       env:
-        WINDOWS_CERTIFICATE_COMMON_NAME: ${{ secrets.WINDOWS_CERTIFICATE_COMMON_NAME }}
+        AZURE_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
+        AZURE_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
+        AZURE_CLIENT_SECRET: ${{ secrets.AZURE_CLIENT_SECRET }}
+        AZURE_CODESIGN_ACCOUNT_NAME: ${{ secrets.AZURE_CODESIGN_ACCOUNT_NAME }}
+        AZURE_CODESIGN_ENDPOINT: ${{ secrets.AZURE_CODESIGN_ENDPOINT }}
+        AZURE_CODESIGN_PROFILE_NAME: ${{ secrets.AZURE_CODESIGN_PROFILE_NAME }}
```

Comment:
Only `AZURE_CODESIGN_PROFILE_NAME` is actually used by this step; please remove the rest: less is more :D

Reply:
They're all required by the PowerShell script to perform the signing.

Reply:
But the powerscript is not called in this step. That is my point :)

Reply:
> But the powerscript is not called in this step. That is my point :)

It should be - the current sign.bat is called from the nightlies [at the moment](https://github.com/OpenTTD/OpenTTD/actions/runs/8275761762/job/22643263613#step:15:1042) in the "Build (without installer)" step.

Reply:
Ah, I see what you mean; meh. It is so bulky now, which makes it hard to read / maintain, so I was hoping we could slim it down. But no, you are right.